.github/workflows/ci.yml

```
@@ -43,7 +43,21 @@ jobs:
     - name: Build PDF
       if: github.repository == 'tldr-pages/tldr' && github.ref == 'refs/heads/main'
       working-directory: ./scripts/pdf
-      run: python render.py ../../pages -c solarized-light
+      run: |
+        for path in ../../pages*; do
+          folder=$(basename "${path}")
+          language="${folder##*.}"
+          case $folder in
+            pages.bn | pages.ja | pages.ko | pages.ml | pages.ta | pages.th | pages.zh | pages.zh_TW)
+              ;;
+            pages)
+              python render.py ${path} -c solarized-light
+              ;;
+            *)
+              python render.py ${path} -c basic -o tldr-pages-${language}.pdf
+              ;;
+          esac
+        done
```

Having played around with this for other bit of review, here's two tweaks:
1. rename the variable from `path` to something else as it ended up breaking my `$PATH` variable which was bad
2. Use `python3` instead of `python` as the latter is more guaranteed to exist on people's systems

```suggestion
        for pageDir in ../../pages*; do
          folder=$(basename "${pageDir}")
          language="${folder##*.}"
          case $folder in
            pages.bn | pages.ja | pages.ko | pages.ml | pages.ta | pages.th | pages.zh | pages.zh_TW)
              ;;
            pages)
              python3 render.py ${pageDir} -c solarized-light
              ;;
            *)
              python3 render.py ${pageDir} -c basic -o tldr-pages-${language}.pdf
              ;;
          esac
        done
```